tests/xtimer_msg/Makefile

```diff
@@ -0,0 +1,8 @@
+APPLICATION = xtimer_msg
+include ../Makefile.tests_common
+
+BOARD_INSUFFICIENT_RAM := stm32f0discovery
```

Comment:
>> > +BOARD_INSUFFICIENT_RAM := stm32f0discovery

The xtimer_msg test is a direct translation of vtimer_msg, the only
changes being the ones necessary to use xtimer instead of vtimer. If
this builds but vtimer_msg doesn't, we have another selling point for
xtimer. :)